### PR TITLE
Add typescript support

### PR DESCRIPTION
**Pull request by keturiosakys**
This update installs and adds the TypeScript plugin to VSCode extension.

### CODE REVIEW

**Comment by flenter** [comment]
This looks ok, but should wait until #5 is merged. After which we should test again if the extension works with the new setup

**Review by arendjr — APPROVED** [review]

File: package-lock.json
Comment (arendjr) [inline package-lock.json:4]: Are you using an older NPM version that caused the lock file version to be lowered?

File: src/extension.ts
Comment (arendjr) [inline src/extension.ts:79]: :muscle:

**Review by flenter — APPROVED** [review]

**Pull request merged**
